lib/src/toggle-group/types.ts

```diff
@@ -0,0 +1,73 @@
+type Space = "xxsmall" | "xsmall" | "small" | "medium" | "large" | "xlarge" | "xxlarge";
+type Margin = {
+  top?: Space;
+  bottom?: Space;
+  left?: Space;
+  right?: Space;
+};
+type SVG = React.SVGProps<SVGSVGElement> | React.FunctionComponent<React.SVGProps<SVGSVGElement>>;
+
+type Option={
+    /**
+    * Number with the option inner value.
+    */
+    value?: string;
+    /**
+    * String with the option display value.
+    */
+    label?: string;
```

Comment:
I consider `label` or `icon` a mandatory prop. I mean, an Option must have one of them, or both, but never any of them. For further understanding: 

![image](https://user-images.githubusercontent.com/44321109/153456125-b5510877-e0e5-4465-90ec-bf784a69dad2.png)

You have an example at the Link component with `icon` and `label` props.

Reply:
Done.